### DISCUSSION TITLE
无法登陆是怎么回事啊 21.1.0这个版本

### DISCUSSION THREAD

**Discussion by Swordnewnewi** [Semporia/TikTok-Unlock#307] [Q&A]
同上

**Comment by Swordnewnewi** [comment]
也无法看评论

**Comment by Swordnewnewi** [comment]
只能刷视频 其他的都做不了![image](https://user-images.githubusercontent.com/118990441/204076674-2e72d390-8dbb-4fbd-ab34-cc2411f796a1.png)

**Comment by Swordnewnewi** [comment]
![image](https://user-images.githubusercontent.com/118990441/204076684-b58dadfb-995e-4175-9a61-5d3e186ab5bb.png)

**Reply by Semporia** [reply]
节点换一下呢？

**Reply by Swordnewnewi** [reply]
> 节点换一下呢？

解决了

**Reply by XH72** [reply]
> > 节点换一下呢？
> 
> 解决了

如何解决的，跟你一样的问题

**Reply by Sadalsund** [reply]
换节点 登陆不了先更新最新版本
